packages/ui-components/src/components/labels-dropdown/labels-dropdown.types.ts

```diff
@@ -0,0 +1,19 @@
+import { EventEmitter } from '@stencil/core';
+import { ISelectMultiOptionsConfig, ISelectMultiOptionsEvents } from '../select-multi-options/select-multi-options.types';
+
+type ExcludedSelectMultiOptionsProps = 'selectionClearable' | 'clearSelectionLabel';
+
+export interface ILabelsDropdown extends Omit<ISelectMultiOptionsConfig, ExcludedSelectMultiOptionsProps> {
+	/** (Optional) Allows outside implementation to open/close dropdown on demand */
+	isOpen?: boolean;
+}
+
+export interface ILabelsDropdownEvents extends ISelectMultiOptionsEvents {
+	/** Emits a signal whenever the 'select all' action is clicked */
+	selectAll: EventEmitter<void>;
```

Comment:
Selection cleared is missing isnt it?

Reply:
inherited from `ISelectMultiOptionsEvents` but fired from a click on a different button